ecosystem/sep-0030.md

```diff
@@ -199,8 +205,22 @@ Name | Type | Description
 `identities` | array | A list of identities that if any one is authenticated will be able to gain control of this account.
 `identities[].role` | string | The role of the identity. This value is not used by the server and is stored and echoed back in responses as a way for a client to know conceptually who each identity represents.
 `identities[].auth_methods` | array | A list of authentication methods that can be used to authenticate as this identity.
-`identities[].auth_methods[].type` | string | The type of identity: `stellar_address`, `phone_number`, `email`, or other.
-`identities[].auth_methods[].value` | string | The identity that if authenticated will be given full permissions of the account, either another stellar address, phone number or email address respectively given the type. When encoding a phone number the ITU-T recommendations [E.123] and [E.164] should be followed. The phone number should be formatted in international notation with a leading `+` as demonstrated in E.123 Section 2.5 to ensure phone numbers are consistently encoded and are globally unique. Spaces should be omitted.
+`identities[].auth_methods[].type` | string | The type of identity. See [Common Authentication Methods].
+`identities[].auth_methods[].value` | string | The identifier or value that uniquely identifies the identity that if authenticated will be given full permissions of the account. For example if the type of authentication is an email address, the value is the email address.
+
+##### Common Authentication Methods
+
+Implementers define the external authentication methods supported by the
+implementation. If an implementation supports any of the following common
+authentication methods the type values below should be used along with the
+specifications for how to format the identifiers to improve interopability
+between clients and servers that may have multiple integraters.
```

Comment:
💡  I like the idea of listing the common authentication methods here. If we're aiming at maximizing interoperability, do you think it's worth suggesting to readers that, if they decide to add a new custom method, they are also highly encouraged to update this SEP with a new entry on the list, so other implementers will also adhere to the standard?

Reply:
I agree with the sentiment, but I don't think that needs to live in the SEP. That's what the discussion thread and elsewhere is for. The SEP probably shouldn't be meta referencing itself and SEP process? Anyone can propose changes to any SEP.

Reply:
Got it. So you think people should update the list as I suggested, but we shouldn't mention this on the SEP text as it would be too meta?

Reply:
That's what I was thinking, but I'm not hard set on that if you think that level of meta is worth it? I was planning to send a message to stellar-dev asking for suggestions for what other formats to include.

Reply:
I think what you're saying makes sense, thinking more about it. It's the same case as for any other interoperability updates: you send a PR, ask for community input, and update the SEP at the end of the process.